web-wallet/README.md

```diff
@@ -53,7 +53,6 @@ VITE_STAKE_CONTRACT="02000000000000000000000000000000000000000000000000000000000
 VITE_TESTNET_NODE="https://nodes.dusk.network/"
 VITE_TESTNET_PROVER_NODE="https://provers.dusk.network/"
 VITE_TRANSFER_CONTRACT="0100000000000000000000000000000000000000000000000000000000000000"
-VITE_GET_QUOTE_API_ENDPOINT="https://api.dusk.network/v1/quote"
```

Comment:
Why don't we just replace this with:
`https://api.coingecko.com/api/v3/coins/dusk-network?community_data=false&developer_data=false&localization=false&market_data=true&sparkline=false&tickers=false`